lib/simp/beaker_helpers.rb

```diff
@@ -374,17 +382,17 @@ def enable_fips_mode_on( suts = hosts )
         on(sut, module_install_cmd)
       end
 
+      # Needed because various versions of crypto-policies are broken
+      on(sut, "puppet resource package crypto-policies ensure=latest") if has_crypto_policies(sut)
```

Comment:
For use in simp-core acceptance tests, this should be part of the `munge_ssh_crypto_policies()`.